src/parser/GeoPoint.h

```diff
@@ -49,19 +46,13 @@ class GeoPoint {
   static constexpr T coordinateMaskLng = (1ULL << numDataBitsCoordinate) - 1;
   static constexpr T coordinateMaskLat = coordinateMaskLng
                                          << numDataBitsCoordinate;
+  static constexpr T coordinateMaskFreeBits =
+      ~(coordinateMaskLat | coordinateMaskLng);
   static constexpr double maxCoordinateEncoded =
-      (double)(1 << numDataBitsCoordinate);
+      (double)((1 << numDataBitsCoordinate) - 1);
```

Comment:
For better readability, can you use the helpers in `BitUtils.h` for these masks, so e.g.
`coordinateMaskLng = bitMaskForLowerBits(numDataBitsCoordinate)`
`coordinateMaskFreeBits = bitMaskForHigherBits(sizeof(T) * 8 - numDataBits)`
`maxCoordinateEncoded = static_cast<double>(coordinateMaskLat)`